src/interactives/interactions/comparing-potential-energy-of-bond.json

```diff
@@ -0,0 +1,328 @@
+{
+  "title": "Comparing Potential Energy of Bond",
+  "publicationStatus": "public",
+  "labEnvironment": "production",
+  "subtitle": "Explore the difference in attractive forces between polar and non-polar molecules.",
+  "category": "",
+  "subCategory": "",
+  "screenshot": "",
+  "aspectRatio": 1.3,
+  "fontScale": 1,
+  "lang": "en-US",
+  "i18nMetadata": "",
+  "helpOnLoad": false,
+  "about": [
+    "Explore the different strengths of dipole-dipole (polar) and London dispersion (non-polar)",
+    "attractions. These intermolecular attractions are shown by dotted lines between the pairs of molecules.",
+    "",
+    "Choose a pair of molecules from the drop-down menu and \"pull\" on the star to separate the",
+    "molecules. Which type of intermolecular attraction is strongest?"
+  ],
+  "theme": "",
+  "showTopBar": true,
+  "showBottomBar": true,
+  "padding": 10,
+  "models": [
+    {
+      "type": "md2d",
+      "id": "instructions",
+      "url": "models/lab-version/1/md2d/interactions/comparing-potential-energy-of-bond/instructions.json",
+      "onLoad": "set('countPotential', 0);",
+      "viewOptions": {
+        "controlButtons": "",
+        "enableKeyboardHandlers": false,
+        "textBoxes": [
+          {
+            "text": "Compare the energy it takes to separate molecules from each other with the energy it takes to separate atoms that are bonded together in a single molecule.",
+            "x": 0.3,
+            "y": 1.7,
+            "fontSize": 0.12,
+            "width": 2.7
+          },
+          {
+            "text": "Choose a pair of molecules from the menu below.",
+            "x": 0.3,
+            "y": 0.4,
+            "fontSize": 0.12,
+            "width": 2.7
+          }
+        ]
+      },
+      "modelOptions": {
+        "useDuration": false
+      }
+    },
+    {
+      "type": "md2d",
+      "id": "npPulling",
+      "url": "models/lab-version/1/md2d/interactions/comparing-potential-energy-of-bond/npPulling.json",
+      "onLoad": [
+                "start();setPairwiseLJProperties(0, 2, { epsilon: 0 }); ",
+                "onDrag('atom',function(){",
+                "callEvery(25, function() {",
+                "var x1 = getAtomProperties(0).x;",
+                "var y1 = getAtomProperties(0).y;",
+                "var x2 = getAtomProperties(3).x;",
+                "var y2 = getAtomProperties(3).y;",
+                "var distx = x1-x2;",
+                "var disty = y1-y2;",
+                "var distance = Math.pow((distx*distx)+(disty*disty),0.5);",
+                "if(distance < 0.4)",
+                "set('countPotential', 5);",
+                "else",
+                "set('countPotential', 5+3*Math.pow(distance,0.5));",
+                "});",
+                "});"
+                ],
```

Comment:
All scripts should be indented as if they were being written in a regular editor. You can see this, for example with the onLoad scripts in your mixing-liquids interactive. It makes it much easier to read.